integration/experiment/gpu_frequency_sweep/gen_gpu_activity_constconfig_recommendation.py

```diff
@@ -63,17 +64,20 @@ def main(full_df):
     The main function. full_df is a report collection dataframe
     """
     df = extract_columns(full_df)
-    gpu_freq_efficient = policy_efficient_energy(df)
-    # This script is not intended to provide an assessment of maximum frequency,
-    # or phi.  As such NAN is provided for these policy values.
-    # The associated agent interprets these as the related system or agent default
-    # config, ex: GPU_FREQ_MAX: NAN --> HW Maximum Frequency, GPU_PHI --> 0.5
-    # (balanced mode).
-    policy = {"GPU_FREQ_MAX" : float('nan'),
-              "GPU_FREQ_EFFICIENT" : gpu_freq_efficient,
-              "GPU_PHI" : float('nan')}
-
-    return policy
+    gpu_freq_efficient = efficient_energy(df)
+
+    json_dict = {"CPU_FREQUENCY_EFFICIENT_HIGH_INTENSITY" :
```

Comment:
Nit regarding format (compared to the format we're following elsewhere in the codebase):
```
{
    "SIGNAL_NAME": {
        ...
    },
    ...
}
```

Same nit applies to other dictionaries below.